lib/foodcritic/rules.rb

```diff
@@ -826,3 +826,28 @@ def invalid_name(ast)
     end
   end
 end
+
+rule 'FC061', 'Valid cookbook version should be defined' do
+  tags %w{metadata}
+  metadata do |ast, filename|
+    ver = ast.xpath(%Q(//command[ident/@value='version']/descendant::tstring_content/@value))
+    if ver.nil? || ver == ''
+      [file_match(filename)]
+    else
+      pass = true
+      parts = ver.to_s.gsub(/\s|'|"/, '').split('.')
+      if parts.length > 1
```

Comment:
Yeah, this is just a rebase of the existing code, I want to add tests and fix that.  CBVs actually have to be exactly 3 dotted decimals and this code can most likely be simplified considerably.